contracts/eden/src/eden-micro-chain.cpp

```diff
@@ -218,6 +219,12 @@ struct status
    uint16_t electionThreshold = 0;
    uint16_t numElectionParticipants = 0;
    uint16_t migrationIndex = 0;
+
+   template <typename T>
+   bool isMigrationCompleted() const
+   {
+      return migrationIndex >= boost::mp11::mp_find<eden::migration_variant, T>::value;
```

Comment:
The of this function is incorrect.  It returns true if the migration has started, not if it has completed.

Reply:
With my above change I think it's fixed, can you please check it again?